src/GitHub/Data/Activities.hs

```diff
@@ -95,7 +95,7 @@ instance Binary Notification
 
 instance FromJSON Notification where
     parseJSON = withObject "Notification" $ \o -> Notification
-        <$> o .: "id"
+        <$> (mkId undefined . read <$> o .: "id")
```

Comment:
what happens here?

Reply:
sorry, not sure what you are asking.

Are you asking about the undefined. I don't think ti should matter since its simply an uneeded (because the type can be inferred in this case) proxy.

Reply:
So then `Proxy` works too, yes?

Reply:
Doesn't original

```haskell
<$> o .: id
```

work? Why?

Reply:
@TomMD ahh, didn't know about Proxy. Seems like it is used pretty widely in this code base. I can make that change.

Reply:
@phadej Are you saying `<$> o .: id` would work even though the value is a string, not an int?